package.json

```diff
@@ -1932,14 +1932,7 @@
                     "default": [
                         {
                             "label": "Compose Down",
-                            "template": "docker-compose ${configurationFile} down",
```

Comment:
Since we now intelligently choose the compose command, we no longer need separate command templates based on context for `compose down`. It is still needed for the two `compose up` commands, because ACI contexts lack a `--build` flag and will error if it is present.